backend/mcp_backend/routers/mcp.py

```
@@ -0,0 +1,963 @@
+
+
+class ConnectionVerificationForm(BaseModel):
+    url: str
+    key: Optional[str] = None
+
+
+@router.post("/verify")
+async def verify_connection(
+    request: Request,
+    form_data: ConnectionVerificationForm,
+    user=Depends(get_admin_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)